src/components/issue-list-item.component.js

```diff
@@ -13,16 +14,28 @@ type Props = {
   locale: string,
 };
 
+const Container = styled.View`
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: center;
+  padding-right: 10;
+  padding: 5;
```

Comment:
Shorthands properties should have units.

Reply:
Fixed